src/main/java/ogc/rs/apiserver/handlers/User.java

```diff
@@ -0,0 +1,64 @@
+package ogc.rs.apiserver.handlers;
+
+import io.vertx.core.json.JsonObject;
+import ogc.rs.apiserver.util.OgcException;
+
+import java.util.UUID;
+
+public class User {
+
+  private UUID userId;
```

Comment:
You can add getters for all the fields, since they need to be accessed by the authZ handlers. Setters may not be required.